src/Avalonia.Controls/TextChangingEventArgs.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Avalonia.Interactivity;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// Provides data specific to a TextChanging event.
+    /// </summary>
+    public class TextChangingEventArgs : RoutedEventArgs
```

Comment:
Note that this is called `TextChangingEventArgs` instead of `TextBoxTextChangingEventArgs` as in WinUI on purpose. It is intended to be generic and used in the future:

1. For other controls such as the AutoCompleteBox
2. Moved to a text control base class like in WPF

Additionally, the name now matches `TextChangedEventArgs` for both WPF and WinUI.